charts/kong/FAQs.md

```diff
@@ -84,7 +84,8 @@ is immutable
 This occurs if a `RELEASE-NAME-kong-init-migrations` Job is left over from a
 previous `helm install` or `helm upgrade`. Deleting it with
 `kubectl delete job RELEASE-NAME-kong-init-migrations` will allow the upgrade
-to proceed.
+to proceed. Setting `migrations.init: false` will prevent the job from spawning
+and creating this issue for future upgrades.
```

Comment:
Question: If `migration.init` is set to `false` during an upgrade, will helm delete the migration job that was created during installation because it no longer sees that resource as part of the release?